pydicom/sequence.py

```diff
@@ -56,12 +57,36 @@ def __init__(self, iterable: Optional[Iterable[Dataset]] = None) -> None:
         # If no inputs are provided, we create an empty Sequence
         super().__init__(validate_dataset, iterable or [])
 
-    def append(self, val: Dataset) -> None:
+    def append(self, val: Dataset) -> None:  # type: ignore[override]
         """Append a :class:`~pydicom.dataset.Dataset` to the sequence."""
         super().append(val)
         val.parent = self._parent
 
-    def insert(self, position: int, val: Dataset) -> None:
+    def extend(self, val: Iterable[Dataset]) -> None:  # type: ignore[override]
+        """Extend a :class:`~pydicom.dataset.Dataset` to the sequence."""
```

Comment:
Added implementation

Reply:
doc string is a little off to me, sounds like only one Dataset.